.npmrc

```diff
@@ -0,0 +1 @@
+package-lock = false
```

Comment:
There is no `package-lock.json`. This prevents automatic `package-lock.json` creation.